modules/bootstrap/README.md

```diff
@@ -9,6 +9,33 @@ The module does *not* configure the bootstrap images, licenses, or configuration
 
 See the examples/vm-series directory.
 
+## Known Limitations
+
+The error message below means that one or more of the keys of the `files` input map is not a name of an existing file.
+Check the key for misspellings. For current versions of Terraform it is not possible to make this message more readable:
+
+```hcl2
+╷
+│ Error: Error in function call
+│
+│   on ../../modules/bootstrap/main.tf line 68, in resource "random_id" "this":
+│   68:     md5 = try(var.files_md5[each.key], md5(file(each.key)))
+│     ├────────────────
+│     │ each.key is "test.txt"
+│     │ var.files_md5 is empty map of string
+│
+│ Call to function "try" failed: no expression succeeded:
+│ - Invalid index (at ../../modules/bootstrap/main.tf:68,28-38)
+│   The given key does not identify an element in this collection value.
+│ - Invalid function argument (at ../../modules/bootstrap/main.tf:68,49-57)
+│   Invalid value for "path" parameter: no file exists at test.txt; this function works only with files that are distributed as part of the configuration source code, so if this file will be created by a resource in this configuration you must instead obtain this result from an attribute of that resource.
+│
+│ At least one expression must produce a successful result.
+╵
+```
+
+If the file does not exist because it is supposed to be generated by the same Terraform run (such as a `local_file` resource), add the hash of its contents to the input `files_md5`.
+
```

Comment:
```suggestion
If the file does not exist because it is supposed to be generated by the same Terraform run (such as a `local_file` resource), add the hash of its contents to the input `files_md5`.

For example `{\"dir/my.txt\" = \"6f7ce3191b50a58cc13e751a8f7ae3fd\"}`"
```